app/javascript/components/lease_ups/LeaseUpApplicationsTableContainer.js

```diff
@@ -64,9 +64,9 @@ const LeaseUpTableContainer = ({
       rowData.address = result.residence_address
     }
 
-    rowData.preference_rank = `${result.preference_record_type} ${result.preference_lottery_rank}`
+    rowData.preference_rank = `${result.preference_record_type} ${result.preference_lottery_rank || result.preference_all_lottery_rank}`
```

Comment:
have we already mapped it so this file doesn't have to know about the all vs not-all rank?

Reply:
pinging again on this comment too

Reply:
removed